feature-ledger-impl/src/main/java/io/novafoundation/nova/feature_ledger_impl/presentation/account/common/bottomSheet/mappers/LedgerDeviceMapper.kt

```diff
@@ -0,0 +1,45 @@
+package io.novafoundation.nova.feature_ledger_impl.presentation.account.common.bottomSheet.mappers
+
+import io.novafoundation.nova.common.resources.ResourceManager
+import io.novafoundation.nova.feature_ledger_api.sdk.device.LedgerDevice
+import io.novafoundation.nova.feature_ledger_api.sdk.device.LedgerDeviceType
+import io.novafoundation.nova.feature_ledger_impl.presentation.account.common.bottomSheet.LedgerMessageCommand
+
+class LedgerDeviceMapper(private val resourceManager: ResourceManager) {
```

Comment:
Maybe we can name it `LedgerDeviceFormatter` to stay consistent with the rest of the presentation formatting naming?
I don't mind naming it `Mapper` as well though